packages/components/src/theme/ComposerTheme/Components/Popover.ts

```diff
@@ -0,0 +1,58 @@
+import { ComponentMultiStyleConfig, cssVar } from '@chakra-ui/react';
+
+const $arrowBg = cssVar('popper-arrow-bg');
+const $backgroundCardColor = cssVar('chakra-colors-background-card');
+
+const baseStyle = {
```

Comment:
I believe we should try to avoid defining textStyles in here, we should try to use the proper component in the UI.

Reply:
I have removed the `textStyle` from the header, but it is required in the `body` part because the body of the Popover could be other than just text, and adding a `Text` with the proper variant would create a strange structure (Text > Button, for example)

Reply:
A `Button` will have his own text style, I don't think we have to define the text style for the body of the popover.